hack/olm-registry/olm-artifacts-template.fedramp.yaml

```diff
@@ -136,6 +136,20 @@ objects:
             - get
             - list
             - watch
+        - apiVersion: rbac.authorization.k8s.io/v1
+          kind: Role
```

Comment:
Adding of these resources is actually subject to if DVO is deployed in Fedramp env as well or not. Am checking internally if it's deployed or not.